src/pages/createCostModelWizard/priceListTable.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  Button,
+  Chip,
+  DataList,
+  DataListAction,
+  DataListCell,
+  DataListItem,
+  DataListItemCells,
+  DataListItemRow,
+  InputGroup,
+  InputGroupText,
+  TextInput,
+  Title,
+  TitleSize,
+  Toolbar,
+  ToolbarGroup,
+  ToolbarItem,
+  ToolbarSection,
+} from '@patternfly/react-core';
+import { SearchIcon } from '@patternfly/react-icons';
+import { EmptyFilterState } from 'components/state/emptyFilterState/emptyFilterState';
+import React from 'react';
+import { CostModelContext } from './context';
+import { label, MetricType, unit } from './metric.utils';
+
+class PriceListTable extends React.Component {
+  public state = { filter: '', current: '' };
+  public render() {
+    return (
+      <CostModelContext.Consumer>
+        {({ tiers, goToAddPL, removeRate }) => {
+          const res = tiers.filter(iter =>
+            `${iter.measurement.toLowerCase()}-${iter.metric.toLowerCase()}`.includes(
+              this.state.filter.toLowerCase()
+            )
+          );
+          return (
+            <>
+              <Title size={TitleSize.xl}>Set a price list</Title>
+              <Title size={TitleSize.md}>
+                The following is a list of rates you have set so far for this
+                price list. To continue adding rates to this price list, click
+                "Add another rate". If you are finished setting this price list,
+                click "Next".
+              </Title>
+              <Toolbar style={{ marginBottom: '10px', marginTop: '10px' }}>
+                <ToolbarSection aria-label="toolbar price list rates top">
+                  <ToolbarGroup>
+                    <ToolbarItem>
+                      <InputGroup>
+                        <TextInput
+                          type="text"
+                          placeholder="Filter by rate..."
+                          value={this.state.current}
+                          onChange={value => {
+                            this.setState({ current: value });
+                          }}
+                          onKeyPress={event => {
+                            if (event.key !== 'Enter') {
+                              return;
+                            }
+                            this.setState({
+                              filter: this.state.current,
+                              current: '',
+                            });
+                          }}
+                        />
+                        <InputGroupText>
+                          <SearchIcon />
+                        </InputGroupText>
```

Comment:
Similar issue as the filter icon. That is, the search icon appears outside the text field, surrounded by a border. The mockup shows something more like this:

<img width="345" alt="Screen Shot 2019-08-16 at 10 45 56 AM" src="https://user-images.githubusercontent.com/17481322/63175967-0c03d480-c013-11e9-984f-a0583e2f3c0d.png">

Reply:
Also want to note that the filter in the cost model page does not have the search icon.